rskj-core/src/main/java/co/rsk/peg/BridgeSerializationUtils.java

```
@@ -63,6 +63,26 @@ private BridgeSerializationUtils() {
         throw new IllegalAccessError("Utility class, do not instantiate it");
     }
 
+    public static byte[] serializeRskAddress(RskAddress rskAddress) {
+        if (rskAddress == null) {
+            throw new IllegalArgumentException("Rsk address cannot be null.");
+        }
+        return RLP.encodeElement(rskAddress.getBytes());
+    }
+
+    public static RskAddress deserializeRskAddress(byte[] rskAddressInBytes) {
+        if (rskAddressInBytes == null || rskAddressInBytes.length == 0) {
+            return null;
```

i think is better to throw an exception here, not returning null

---

I already tried, but it is not possible because the repository passes an empty array when no entry is found. This is used by the `getXxx` method when there is no entry for that key, so it is expecting a null. 

Something we can do is to return an optional instead.